Storage/ThirdParty/SwiftData.swift

```diff
@@ -1015,7 +1015,7 @@ open class ConcreteSQLiteDBConnection: SQLiteDBConnection {
                 Sentry.shared.sendWithStacktrace(message: "SQLITE_CORRUPT", tag: SentryTag.swiftData, severity: .error, description: "DB file '\(filename)'. \(error.localizedDescription)")
             }
 
-            let message = "Error code: \(error.code), \(error) for SQL \(String(sqlStr.prefix(500)))."
+            let message = "Error code: \(error.code), \(error) for SQL \(sqlStr)."
```

Comment:
Careful. This was here to avoid dumping very large queries in the log _and sending them to Sentry_ — consider that we sometimes have `VALUES (?, 1, 1), (?, 1, 1)` clauses 999 items long, resulting in 15KB SQL strings.

Reply:
Good catch. I forgot to revert this before pushing.